src/slic3r/GUI/BitmapComboBox.cpp

```diff
@@ -90,72 +79,6 @@ BitmapComboBox::~BitmapComboBox()
 {
 }
 
-#ifdef __APPLE__
```

Comment:
MacOS build failure are caused by removing these two functions

Reply:
Looks like PS had meant for the declarations to be removed too but I just overlooked it. About to push a fix.